esbuild/copy.js

```diff
@@ -0,0 +1,47 @@
+const { mkdir, copyFile } = require( 'fs' ).promises;
+const { dirname } = require( 'path' );
+const { getFiles, copyAll } = require( './utils.js' );
+const { unprocessed, modules } = require( '../config/environment.js' ).paths;
+
+module.exports = async function( baseConfig ) {
+  const files = await getFiles( unprocessed );
+  const staticFiles = files.filter( v => !v.match( /.js$|.less$|.css$/ ) );
+
+  const inDirs = [ ...new Set( staticFiles.map( v => dirname( v ) ) ) ];
+  const outDirs = [
+    ...inDirs.map( v => v.replace( unprocessed, baseConfig.outdir ) ),
+    // Create specific icon directory
+    `${ baseConfig.outdir }/icons`,
+    // Hande prebuilt lightbox dep
+    ...[ '', '/images', '/js', '/css' ]
+      .map( v => `${ baseConfig.outdir }/lightbox2${ v }` )
```

Comment:
Per #6983 you no longer have to worry about all the `lightbox2` assets in this file.